db.js

```diff
@@ -149,7 +149,19 @@ var friends = (function () {
 
 var users = (function () {
   return {
-    get: function (userId) {
+    getUserByName: function (userName) {
+      var expr = "'" + userName + "%'";
+      var sql = "SELECT * FROM users WHERE name LIKE " + expr;
```

Comment:
You can use backticks here instead of quotes to inject code in there. Like:
`` `SELECT * FROM users WHERE name LIKE '${userName}%'` ``

Reply:
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals